Doc/library/sys.rst

```diff
@@ -449,10 +449,12 @@ always available.
 
 .. function:: exit([arg])
 
-   Exit from Python.  This is implemented by raising the :exc:`SystemExit`
+   Exit from Python.  If called in the main thread, this raises the :exc:`SystemExit`
    exception, so cleanup actions specified by finally clauses of :keyword:`try`
    statements are honored, and it is possible to intercept the exit attempt at
-   an outer level.
+   an outer level. When called from a thread other than the main thread, this
+   causes the thread to exit silently, nothing is printed and is equivalent to calling
+   :func:`thread.exit`.
```

Comment:
Have you verified this (through testing or from the code)?

Reply:
I tried with a sample code. Exception is raised from both main and non-primary thread. The doc is updated. Please review.

Reply:
The edit looks good. Thanks.